index.html

```diff
@@ -301,6 +301,21 @@ <h3>
           </table>
         </section>
 
+        <section id="normative">
+          <h3>
+            Registries
+          </h3>
+          <p>
+The Working Group may create a set of
+<a href="https://www.w3.org/2021/Process-20211102/#registries">registries</a>
+including <a href="https://www.w3.org/2021/Process-20211102/#registry-definition">registry definitions</a>
+and <a href="https://www.w3.org/2021/Process-20211102/#registry-table">registry tables</a>
+to support extension points in the above normative deliverables. Registries for
+extension points that are required by any of the above normative deliverables
+must have at least one standardized entry.
```

Comment:
```suggestion
to support extension points in the above normative deliverables. Registries for
extension points that are required by any of the above normative deliverables
must have at least one standardized entry.
```

-1, I expect that we'd have to remove features in the VC spec and that would be controversial.

Reply:
Hrm, on second read, I'm confused. So, existing extension points that don't have normative references would still be allowed?

Reply:
I tried to add the language that seemed to have the most consensus during the WG call on March 2nd.
Currently, the only normatively required extension point in the VC data model is 'proof', which we will standardize in the data integrity work. I don't see any features that we would need to remove. What am I missing?

Reply:
> I don't see any features that we would need to remove. What am I missing?

We currently don't have normative deliverables in the current charter defined for any of the following extension points:

https://www.w3.org/TR/vc-data-model/#status
https://www.w3.org/TR/vc-data-model/#evidence
https://www.w3.org/TR/vc-data-model/#terms-of-use
https://www.w3.org/TR/vc-data-model/#refreshing
https://www.w3.org/TR/vc-data-model/#data-schemas

One reading of the language wrt. "normative registry entries" is that every single one of those features should be removed in the VC 2.0 work, which I expect would lead to objections from organizations relying on those extension points out in the field.

Reply:
none of those are required by the VC data model, therefore none of those would need a standardized entry in a registry.
But also, the language has been removed form this PR and moved to #101

Reply:
> none of those are required by the VC data model, therefore none of those would need a standardized entry in a registry.

Ah, I see, and am now slowly remembering the conversation... I'm not opposed IF everyone reads the language in the same way. I'll try providing some changes in PR #101 to try to make it clear that it only applies to normative properties... though, I expect, this wouldn't address @jyasskin's concern.